Sources/Down/AST/Visitors/AttributedStringVisitor.swift

```diff
@@ -20,17 +20,23 @@ public class AttributedStringVisitor {
 
     private let styler: Styler
     private let options: DownOptions
+    private let listPrefixGeneratorBuilder: ListItemPrefixGeneratorBuilder
     private var listPrefixGenerators = [ListItemPrefixGenerator]()
 
     /// Creates a new instance with the given styler and options.
     ///
     /// - parameters:
     ///     - styler: used to style the markdown elements.
     ///     - options: may be used to modify rendering.
+    ///     - listPrefixGeneratorBuilder: may be used to modify list prefixes.
 
-    public init(styler: Styler, options: DownOptions = .default) {
+    public init(
+        styler: Styler,
+        options: DownOptions = .default,
+        listPrefixGeneratorBuilder: ListItemPrefixGeneratorBuilder = StaticListItemPrefixGeneratorBuilder()) {
```

Comment:
For formatting consistency:

```suggestion
    public init(
        styler: Styler,
        options: DownOptions = .default,
        listPrefixGeneratorBuilder: ListItemPrefixGeneratorBuilder = StaticListItemPrefixGeneratorBuilder()
    ) {
```

Reply:
Will do!